db/db.go

```diff
@@ -0,0 +1,58 @@
+package db
+
+//db global storage in memory
+var db = make(map[int64]interface{})
+
+//id auto incremental value
+var index int64 = 0
+
+//Entity is an interface that allows to interact with the database
```

Comment:
Entity is not actually an interface that you are defining. :scream: 

Here Entity is a type conformed by a simple empty struct and It has several methods defined such as Create, Update, etc. 

There is one key thing missing to have a proper Entity Interface. I can definitely help you out if you need it :nerd_face: